### PR TITLE
Fix BuildKite publishing of source_model

### PR DESCRIPTION
**Pull request by alexwlchan**
The previous commit to main published source_model 4268, but didn't publish any version of internal_model.

Because source_model has a dependency on internal_model, when you try to load that in the catalogue-api repo, you get an error:

> I need internal_model 4268 to go with source_model 4268, but I can't find it. 💥 

See the BuildKite failures for https://github.com/wellcomecollection/catalogue-api/pull/105. Publishing both libraries if either of them has changed will get around this for now.

This isn't an issue in scala-libs because publishing there is an all-or-nothing process – either everything gets published, or nothing does.

For https://github.com/wellcomecollection/catalogue-api/issues/103

### CODE REVIEW

File: common/internal_model/src/main/scala/weco/catalogue/internal_model/locations/AccessStatus.scala [hunk 1]
```diff
@@ -24,10 +24,10 @@ sealed trait AccessStatus extends EnumEntry { this: AccessStatus =>
   }
 }
 
-object AccessStatus extends Enum[License] {
+object AccessStatus extends Enum[AccessStatus] {
```
Comment (jamesgorrie) [inline common/internal_model/src/main/scala/weco/catalogue/internal_model/locations/AccessStatus.scala:27]: Was this meant to slip in here?

Comment (alexwlchan) [reply common/internal_model/src/main/scala/weco/catalogue/internal_model/locations/AccessStatus.scala:27]: Oops. It was not. I can pull it into a separate PR if you think it's important, but it's just a typo I spotted while working on this and I forgot to pull into a separate PR.

Comment (jamesgorrie) [reply common/internal_model/src/main/scala/weco/catalogue/internal_model/locations/AccessStatus.scala:27]: It looks like more than a typo? If it's a noop, all good to stay here.

**Review by jamesgorrie — APPROVED** [review]

File: common/internal_model/src/main/scala/weco/catalogue/internal_model/locations/AccessStatus.scala (hunk 1 above)
Comment (alexwlchan) [reply common/internal_model/src/main/scala/weco/catalogue/internal_model/locations/AccessStatus.scala:27]: typo or copy/paste error. Should be a no-op, but CI will confirm.

**Pull request merged**